docs/conf.py

```diff
@@ -30,19 +30,14 @@
 from docutils.utils import get_source_line
 
 
-def _warn_node(self, msg, node):
-    """Do not warn on external images."""
-    if not msg.startswith('nonlocal image URI found:'):
-        self._warnfunc(msg, '{0}:{1}'.format(get_source_line(node)))
-
-sphinx.environment.BuildEnvironment.warn_node = _warn_node
-
-
 # -- General configuration ------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
 #needs_sphinx = '1.0'
 
+# Do not warn on external images.
```

Comment:
@jirikuncar In theory, the ideal situation is obviously not to pin anything, and fix conflicts as they come.  In practice, though, the timing for fixing conflicts is not the best regarding the current busy times with ongoing Zenodo 3 sprint... Especially for "non-functional" dependencies such as Sphinx, where living with an older pinned version may be a good practical compromise for some time to come.

That said, I'm all for fixing it everywhere, if people are willing to donate the extra time to follow good principles :smile: 
 
```console
$ grep Sphinx.*1.3 ./src/*/{.travis.yml,setup.py} | awk -F: '{print $1;}' | xargs dirname | awk -F/ '{print $NF;}' | sort -u
analysis-preservation.cern.ch
datacite
dictdiffer
dojson
domapping
flask-cli
idutils
invenio-accounts
invenio-annotations
invenio-assets
invenio-base
invenio-celery
invenio-classifier
invenio-cloudconnector
invenio-comments
invenio-communities
invenio-config
invenio-db
invenio-documents
invenio-ext
invenio-formatter
invenio-groups
invenio-i18n
invenio-jsonschemas
invenio-knowledge
invenio-logging
invenio-marc21
invenio-memento
invenio-oauth2server
invenio-pidstore
invenio-previewer
invenio-records
invenio-records-ui
invenio-redirector
invenio-rest
invenio-search
invenio-sequencegenerator
invenio-tags
invenio-testing
invenio-theme
invenio-unapi
invenio-upgrader
invenio-userprofiles
invenio-utils
invenio-workflows
jsonresolver
requirements-builder
timegate
workflow